content/docs/create-a-new-react-app.md

```diff
@@ -8,87 +8,87 @@ prev: add-react-to-a-website.html
 
-This is also **the easiest way to integrate React into an existing website.** You can always add a larger toolchain if you find it helpful!
+Cara tersebut juga merupakan **cara termudah untuk mengintegrasikan *React* ke dalam *website* yang sudah ada.** Anda bisa menambah *Toolchains* yang lebih besar kapanpun anda inginkan jika anda merasa itu dapat membantu!
 
-## Recommended Toolchains {#recommended-toolchains}
+## *Toolchains* Yang Direkomendasikan {#recommended-toolchains}
 
-The React team primarily recommends these solutions:
+Tim *React* sangat merekomendasikan solusi ini:
 
-- If you're **learning React** or **creating a new [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app).
-- If you're building a **server-rendered website with Node.js,** try [Next.js](#nextjs).
-- If you're building a **static content-oriented website,** try [Gatsby](#gatsby).
-- If you're building a **component library** or **integrating with an existing codebase**, try [More Flexible Toolchains](#more-flexible-toolchains).
+- Jika anda **Sedang belajar *React*** atau **membuat Aplikasi [single-page](/docs/glossary.html#single-page-application) baru,** gunakan [*Create React App*](#create-react-app).
+- Jika anda sedang membangun sebuah ***website server-rendered* menggunakan Node.js,** coba gunakan [Next.js](#nextjs).
+- Jika anda sedang membangun sebuah ***website* statis berorientasi konten,** coba gunakan [Gatsby](#gatsby).
+- Jika anda sedang membangun sebuah ***library* komponen** atau **menggabungkannya dengan *codebase* yang sudah ada,** coba gunakan [*toolchains* yang lebih fleksibel](#more-flexible-toolchains).
 
-### Create React App {#create-react-app}
+### *Create React App* {#create-react-app}
 
-[Create React App](http://github.com/facebookincubator/create-react-app) is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](/docs/glossary.html#single-page-application) application** in React.
+[*Create React App*](http://github.com/facebookincubator/create-react-app) adalah sebuah *environment* yang nyaman untuk **mempelajari *React***, dan *environment* ini merupakan cara terbaik untuk mulai membangun **sebuah aplikasi [single-page](/docs/glossary.html#single-page-application)** di *React*.
 
-It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimizes your app for production. You’ll need to have Node >= 6 and npm >= 5.2 on your machine. To create a project, run:
+*Environment* ini mengatur *environment* pengembangan anda. Sehingga, anda dapat menggunakan fitur terbaru *JavaScript*, menyediakan pengalaman pengembang yang menyenangkan, dan mengoptimalkan aplikasi anda untuk produksi. Anda akan membutuhkan *Node* >= 6 dan *npm* >= 5.2 di komputer anda. Untuk membuat suatu proyek baru, jalankan:
```

Comment:
```suggestion
Create React App mengatur lingkungan pengembangan anda. Sehingga, Anda dapat menggunakan fitur terbaru *JavaScript*, memberikan pengalaman pengembangan yang menyenangkan, dan mengoptimalkan aplikasi anda untuk lingkungan produksi. Anda akan membutuhkan *Node* >= 6 dan *npm* >= 5.2 di komputer anda. Untuk membuat suatu proyek baru, jalankan:
```

What do you think? 😄

Reply:
`It sets up your development environment so that you can use the latest JavaScript features,`

I think the first `It` in this sentence refer to the Create React App and not the `development environment`, CMIIW 😄

Reply:
yes. i thought so. `lingkungan` is refer to `create react app` since `create react app` is defined as an _environment_ in the first paragraph. But, i think i will replace it with 'create react app' anyway since it is easier to understand. Thank you for your correction, again. 😄 i'll fix it soon.